### PR TITLE
Relative links created, Page not found fixes, Reduntant links of YT removed, sidebar integrated with Plugins 

### LINKED ISSUE #540: Bug Report: A. page not found fixes B. Redundant link fixes  C. adding "Functionalities page" to Sidebar
**Describe the bug**
Here, https://docs.talawa.io/docs/internships/internship-ideas/#api---primary-idea-areas

A. The Github Extenship links lead to "page not found" errors, and hence should be removed
image

B. The YouTube inspiration link 2 and 4 is the same
image

C. The Plugin guide redirects to a functionalities page, which isn't added to the Sidebar and hence is unreachable otherwise.
image
image

D. Apart from this, I also plan to make all the links relative, wherever possible.

**Expected behavior**
I wish to resolve this, by making the necessary changes.

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Nitya-Pasrija**
**What kind of change does this PR introduce?**
I have implemented relative links and resolved issues with "Page not found" errors. Additionally, I have removed redundant links to YouTube and integrated the sidebar with the plugin docs. 

**Issue Number:**

Fixes #540 

**Snapshots/Videos:**
image
image
image

(https://docs.talawa.io/docs/internships/internship-ideas)

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**
Yes

P.S - Continued from PR #551 
Conflicts resolved

### CODE REVIEW

**Comment by Nitya-Pasrija** [comment]
@Cioppolo14 
Have resolved the conflicts now :)
Apologies for the delay. I had to update my cloned repository with some recent changes, after that the conflicts were successfully resolved.

**Comment by salman-farooqi** [comment]
Hello Nitya, I think notifications for your slack are turned off, I had a question related to one of your previous PRs https://github.com/PalisadoesFoundation/talawa-docs/pull/517. I'm also given a task related to the same problem, so kindly if you clear my doubts on that. I have a DM in your slack. Thanks!

**Comment by Nitya-Pasrija** [comment]
Surely @salman-farooqi 
Just saw your DM

File: docs/internships/ideas.md [hunk 1]
```diff
@@ -39,13 +39,13 @@ Many of you reading this page are interested in participating in our various int
 
 1. Make sure to read the relevant `Introduction` and `Applying` pages first.
    1. GitHub Externship:
-      1. [Introduction](https://docs.talawa.io/docs/internships/github/github-introduction).
-      1. [Application Guide](https://docs.talawa.io/docs/internships/github/github-application-process).
+      1. Introduction
```
Comment (Cioppolo14) [inline docs/internships/ideas.md]: The github externship section can be completely removed.

**Review by Cioppolo14 — CHANGES_REQUESTED** [review]
Did the relative links work during testing? I know there have been previous issues.

**Comment by salman-farooqi** [comment]
> Did the relative links work during testing? I know there have been previous issues.

Yes, it works. I've checked Nitya's previous PR also where she has used both relative and absolute links.
Absolute links: https://docs.talawa.io/docs/features/features-introduction
Relative links: https://docs.talawa.io/docs/features/members/members-overview

**Comment by Nitya-Pasrija** [comment]
@Cioppolo14
After conferring with Salman, I discovered that utilizing relative references such as './features/file.md' results in the link being opened in the current tab, whereas the use of absolute references like 'https://talawa.io/docs/features/file.md' causes the link to be opened in a new tab.

> > Did the relative links work during testing? I know there have been previous issues.
> 
> Yes, it works. I've checked Nitya's previous PR also where she has used both relative and absolute links. Absolute links: https://docs.talawa.io/docs/features/features-introduction Relative links: https://docs.talawa.io/docs/features/members/members-overview

File: docs/internships/ideas.md (hunk 1 above)
Comment (Nitya-Pasrija) [reply docs/internships/ideas.md]: Will do :)
Also, should I add Outreachy in this list, and link it's introduction and contribution page?

Comment (Cioppolo14) [reply docs/internships/ideas.md]: @Nitya-Pasrija Yes, please! Good point.

**Review by Cioppolo14 — APPROVED** [review]

**Pull request merged**